Version 1.9.0 testing as replacement in CXF soap project gives this runtime error

```
Caused by: javax.wsdl.WSDLException: WSDLException: faultCode=CONFIGURATION_ERROR: No Java extensionType found to represent a '{http://www.w3.org/2001/XMLSchema}schema' element in the context of a 'javax.wsdl.Types'.
	at javax.wsdl.extensions.ExtensionRegistry.createExtension(ExtensionRegistry.java:231) ~[libre-wsdl4j-1.9.0.jar:na]
	at org.apache.cxf.wsdl11.ServiceWSDLBuilder.getSchemaImplementation(ServiceWSDLBuilder.java:284) ~[cxf-rt-wsdl-3.5.3.jar:3.5.3]
```

--- a/src/main/java/com/ibm/wsdl/extensions/PopulatedExtensionRegistry.java
+++ b/src/main/java/com/ibm/wsdl/extensions/PopulatedExtensionRegistry.java
@@ -408,11 +408,6 @@ public class PopulatedExtensionRegistry extends ExtensionRegistry
                       MIMEConstants.Q_ELEM_MIME_MIME_XML,
                       MIMEMimeXmlImpl.class);
 
-    // [tellef - 2019-02-22]: The following code has been removed
-    //  to prevent WSDL4j from fetching external files using HTTP.
-    //
-    //Register the schema parser
-    /*
     mapExtensionTypes(Types.class, SchemaConstants.Q_ELEM_XSD_1999,
         SchemaImpl.class);
     registerDeserializer(Types.class, SchemaConstants.Q_ELEM_XSD_1999,
@@ -433,6 +428,5 @@ public class PopulatedExtensionRegistry extends ExtensionRegistry
         new SchemaDeserializer());
     registerSerializer(Types.class, SchemaConstants.Q_ELEM_XSD_2001,
         new SchemaSerializer());
-    */
   }
 }

